pkg/sink/sink.go

```diff
@@ -72,27 +72,39 @@ func (r Resource) HandleEvent(response http.ResponseWriter, request *http.Reques
 	log.Printf("EventListener: %s in Namespace: %s handling event (EventID: %s) with payload: %s and header: %v",
 		r.EventListenerName, r.EventListenerNamespace, eventID, string(event), request.Header)
 
+	result := make(chan int, 10)
```

Comment:
Why is the buffer size 10?

Reply:
There is no reason :-).  Any number should be OK.  It should work fine with 1 but but it's a little nice not to make some thread wait for the queue.

Reply:
nit: If we don't want those goroutines to wait, shouldn't the buffer be `len(el.Spec.Triggers)`?

Also, I don't think it really matters if the `executeTrigger()` goroutines are blocked by the channel, because they all return/exit right after sending the message to the channel. That said, I guess it can only help to have the buffer.